server.js

```diff
@@ -1,13 +1,18 @@
 import express from "express";
 import cors from "cors";
+import path from "path";
+import avocadoSalesData from "./data/avocado-sales.json";
```

Comment:
you don't need avocado sales data, do you? you could also comment out the other links to json files and delete them to to make your code neater (line 9-14)